donkeycar/parts/keras.py

```diff
@@ -143,20 +130,20 @@ def run(self, img_arr):
         img_arr = img_arr.reshape((1,) + img_arr.shape)
         angle_binned, throttle = self.model.predict(img_arr)
         N = len(throttle[0])
-        throttle = dk.utils.linear_unbin(throttle, N=N, offset=0.0, R=self.throttle_range)
+        throttle = dk.utils.linear_unbin(throttle, N=N,
+                                         offset=0.0, R=self.throttle_range)
         angle_unbinned = dk.utils.linear_unbin(angle_binned)
         return angle_unbinned, throttle
-    
-    
+
     
 class KerasLinear(KerasPilot):
-    '''
-    The KerasLinear pilot uses one neuron to output a continous value via the 
+    """
+    The KerasLinear pilot uses one neuron to output a continous value via the
     Keras Dense layer with linear activation. One each for steering and throttle.
     The output is not bounded.
-    '''
-    def __init__(self, num_outputs=2, input_shape=(120, 160, 3), roi_crop=(0, 0), *args, **kwargs):
-        super(KerasLinear, self).__init__(*args, **kwargs)
+    """
+    def __init__(self, num_outputs=2, input_shape=(120, 160, 3), roi_crop=(0, 0)):
```

Comment:
Lets get rid of `roi_crop` because that won't compose well with more complicated transforms made possible by the new datastore. 

We should also document this new workflow.

Reply:
I'll remove the `roi_crop` and update the PR.